examples/platform/nxp/se05x/DeviceAttestationSe05xCredsExample_v2.cpp

```diff
@@ -178,108 +180,77 @@ CHIP_ERROR ExampleSe05xDACProviderv2::SignWithDeviceAttestationKey(const ByteSpa
 
     msg_reader.Init(message_to_sign);
 
-    /* To be removed. Use common key id to sign message */
-    static bool sign_cert_decl_attest = 1;
-
-    if (sign_cert_decl_attest)
-    {
-        /* Check if certificate declaration tag is present and Skip certificate declaration tag */
-        ReturnErrorOnFailure(TLV::Utilities::Find(msg_reader, TLV::ContextTag(1), tagReader));
-
-        ReturnErrorOnFailure(TLV::Utilities::Find(msg_reader, TLV::ContextTag(2), tagReader));
-        uint8_t attlen = tagReader.GetLength();
-        VerifyOrReturnError(attlen > 0, CHIP_ERROR_INVALID_TLV_TAG);
-        /* Get attestation nonce */
-        ByteSpan attest_nonce;
-        ReturnErrorOnFailure(tagReader.Get(attest_nonce));
-        /* Set attestation nonce */
-        VerifyOrReturnError(CHIP_NO_ERROR ==
-                                se05xSetCertificate(CD_ATTEST_NONCE_DATA_SE05X_ID, attest_nonce.data(), attest_nonce.size()),
-                            CHIP_ERROR_INTERNAL);
+    uint16_t taglen = 0;
+    ByteSpan tagvalue;
+    uint8_t tempBuf[2] = {
+        0,
+    };
 
-        ReturnErrorOnFailure(TLV::Utilities::Find(msg_reader, TLV::ContextTag(3), tagReader));
-        uint8_t tslen = tagReader.GetLength();
-        if (tslen > 0)
-        {
-            ByteSpan time_stamp;
-            ReturnErrorOnFailure(tagReader.Get(time_stamp));
-            /* Set time stamp data */
-            VerifyOrReturnError(CHIP_NO_ERROR ==
-                                    se05xSetCertificate(CD_TIME_STAMP_DATA_SE05X_ID, time_stamp.data(), time_stamp.size()),
-                                CHIP_ERROR_INTERNAL);
-        }
-        /* Set time stamp length */
-        VerifyOrReturnError(CHIP_NO_ERROR == se05xSetCertificate(CD_TIME_STAMP_LEN_SE05X_ID, &tslen, 1), CHIP_ERROR_INTERNAL);
+    tempBuf[0] = (uint8_t) TLV::TLVElementType::Structure;
+    VerifyOrReturnError(CHIP_NO_ERROR == se05xSetCertificate(START_CONTAINER_SE05X_ID, tempBuf, 1), CHIP_ERROR_INTERNAL);
 
-        if ((tagReader.GetRemainingLength() + 1 /* End container */) >= 16)
+    for (int i = 1; i <= 3; i++)
+    {
+        if (TLV::Utilities::Find(msg_reader, TLV::ContextTag(i), tagReader) == CHIP_NO_ERROR)
```

Comment:
And if it's not CHIP_NO_ERROR, this might silently be ignoring broken TLV and proceeding to sign something anyway.

Reply:
Updated the code to handle the tlv errors.